Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -182,7 +182,12 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     // Log share initiated event
     [self userCompletedAction:BNCShareInitiatedEvent];
     UIActivityItemProvider *itemProvider = [self getBranchActivityItemWithLinkProperties:linkProperties];
-    NSMutableArray *items = [NSMutableArray arrayWithObject:itemProvider];
+    NSMutableArray *items;
+    if (itemProvider) {
```

Comment:
Good point. I would like to be able to reproduce this before making this change.